modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpLaravelServerCodegen.java

```diff
@@ -0,0 +1,286 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openapitools.codegen.languages;
+
+import io.swagger.models.properties.*;
+import io.swagger.v3.oas.models.*;
+import io.swagger.v3.oas.models.media.*;
+import io.swagger.v3.oas.models.parameters.*;
+import org.openapitools.codegen.*;
+
+import java.io.File;
+import java.util.*;
+
+import io.swagger.v3.oas.models.media.*;
+import io.swagger.v3.oas.models.PathItem;
+import io.swagger.v3.oas.models.PathItem.HttpMethod;
+import io.swagger.v3.oas.models.*;
+import io.swagger.v3.oas.models.parameters.*;
+import io.swagger.v3.core.util.Yaml;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class PhpLaravelServerCodegen extends AbstractPhpCodegen {
+    @SuppressWarnings("hiding")
+    protected String apiVersion = "1.0.0";
+    protected String variableNamingConvention = "camelCase";
+
+    /**
+     * Configures the type of generator.
+     *
+     * @return the CodegenType for this generator
+     * @see org.openapitools.codegen.CodegenType
+     */
+    public CodegenType getTag() {
+        return CodegenType.SERVER;
+    }
+
+    /**
+     * Configures a friendly name for the generator.  This will be used by the generator
+     * to select the library with the -g flag.
+     *
+     * @return the friendly name for the generator
+     */
+    public String getName() {
+        return "php-laravel";
+    }
+
+    /**
+     * Returns human-friendly help for the generator.  Provide the consumer with help
+     * tips, parameters here
+     *
+     * @return A string value for the help message
+     */
+    public String getHelp() {
+        return "Generates a PHP laravel server library.";
+    }
+
+    /**
+     * Class constructor
+     */
+    public PhpLaravelServerCodegen() {
+        super();
+
+        embeddedTemplateDir = templateDir = "php-laravel";
+
+        /*
+         * packPath
+         */
+        invokerPackage = "php-laravel";
+        packagePath = "";
+        outputFolder = packagePath + File.separator + srcBasePath;
+
+        /*
+         * Api Package.  Optional, if needed, this can be used in templates
+         */
+        apiPackage = "app.Http.Controllers";
+
+        /*
+         * Model Package.  Optional, if needed, this can be used in templates
+         */
+        modelPackage = "models";
+
+        // template files want to be ignored
+        modelTemplateFiles.clear();
```

Comment:
@renepardon please remove this line to have the model files auto-generated.

cc @ackintosh